src/object/authenticate_context.cpp

```diff
@@ -256,6 +256,10 @@ authenticate_context::login (const char *name, const char *password, bool ignore
 	    }
 	  strcpy (user_name, name);
 	}
+      else
+	{
+	  strcpy (user_name, "");
```

Comment:
```suggestion
	  user_name[0] = '\0';
```

함수 호출 대신 바로 초기화할 수 있을 것 같습니다.
```
// authenticate_context.hpp:141
char user_name[DB_MAX_USER_LENGTH + 4] = { '\0' };
```